pages/index.tsx

```diff
@@ -62,10 +66,48 @@ export default function Home() {
             ))}
           </ButtonGroup>
         </SectionContainer>
+        <SectionContainer sx={{ margin: 'auto', maxWidth: '65rem' }} maxWidth={false}>
```

Comment:
I think that we might want to remove the maxWidth here, it looks like it is causing this paper section to be a different size than the rest on desktop screens.

Reply:
I see what you mean - I removed the maxWidth and margin properties